scripts/Source/sslThreadModel.psc

```diff
@@ -542,6 +493,56 @@ state Making
 			endIf
 		endIf
 		
+		; ------------------------- ;
```

Comment:
Moved down on the scripts to first validate the animation list and the common tags since some of the CenterRef functions depend of the animations tags like Aggressive.